src/App.tsx

```diff
@@ -1,16 +1,37 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [movie, setMovie] = useState(moviesFromServer);
+
+  const addMovie = (
+    title: string,
+    description: string,
+    imgUrl: string,
+    imdbUrl: string,
+    imdbId: string,
```

Comment:
you could pass whole movie instead of separate properties
```suggestion
newMovie
```